scripts/machine_translation/README.md

```diff
@@ -1,170 +1,328 @@
 # Machine Translation
 
+To run the scripts, you are recommended to install [tensorboardX](https://github.com/lanpa/tensorboardX).
+
+```
+python3 -m pip install tensorboardX
```

Comment:
out of curiosity, is tensorboardx still necessary for using tensorboard without tensorflow?

Reply:
Not sure. I haven't checked the latest version of Tensorboard. But in the official documentation of Tensorboard, there is only the use-case within TF.

Reply:
tensorboard should be ok now.